include/mbedtls/md.h

```diff
@@ -302,6 +302,7 @@ int mbedtls_md_update( mbedtls_md_context_t *ctx, const unsigned char *input, si
  *
  * \param ctx       The generic message-digest context.
  * \param output    The buffer for the generic message-digest checksum result.
+ *                  This must not be \c NULL.
```

Comment:
We should be more precise here and document how much space we need. Suggestion: "This must be a writable buffer large enough to hold the message-digest. See for example mbedtls_md_get_size()."